src/main/client/src/dashBoard/DashBoard.js

```diff
@@ -6,6 +6,8 @@ import DeleteBookModal from "../common/modals/DeleteBookModal";
 import { connect } from "react-redux";
 import * as bookListActions from "./bookList/reducer";
 import { bindActionCreators } from "redux";
+import { Button } from "semantic-ui-react";
+import ScottLogicLogo from "../common/ScottLogicLogo.jpg";
```

Comment:
This shouldn't have a capital letter for the variable name

Reply:
changed